scripts/build

```diff
@@ -94,6 +95,13 @@ if [ "$VERBOSE" -ge 1 ]; then
     sed -i -e 's/rpmbuild/rpmbuild --quiet/' "$DIST_SRC"/Makefile*
     MAKE_OPTS="$MAKE_OPTS -s"
 fi
+# Build the rpmdb again, in case of huge rpm version difference that makes
+# rpmdb --rebuilddb doesn't work anymore. Export using rpm from outside
+# chroot and import using rpm from within chroot
+sudo rpmdb --root="$CHROOT_DIR" --exportdb > "$CACHE_DIR"/rpmdb.export
```

Comment:
This needs to check if the export succeeded, otherwise we'll have another https://github.com/QubesOS/qubes-issues/issues/6651